rusk/src/lib/services/prover/handler.rs

```diff
@@ -0,0 +1,132 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+//
+// Copyright (c) DUSK NETWORK. All rights reserved.
+
+use super::{rusk_proto, ServiceRequestHandler};
+use dusk_bytes::Serializable;
+
+use dusk_plonk::prelude::*;
+use dusk_wallet_core::UnprovenTransaction;
+use lazy_static::lazy_static;
+use rusk_profile::keys_for;
+use rusk_proto::{ProverRequest, ProverResponse};
+use tonic::{Request, Response, Status};
+use transfer_circuits::{
+    CircuitInput, CircuitInputSignature,
+    ExecuteCircuit::{self, *},
```

Comment:
I would avoid to use this syntax here, specially because the `*` seems to be used only inside the `circuit_from_numbers`. Looks more clean if you just import `ExecuteCircuit` here and then `use ExecuteCircuit` in the function.

Reply:
Right!